src/Inventory/CreateList.js

```diff
@@ -20,21 +21,38 @@ class CreateList extends React.Component {
 
 	constructor(props) {
 		super(props);
+
 		this.state = {
-			name: ''
+			name: ""
 		};
 	}
 
 	_createNewList() {
+
+		// if no name just go back without creating the list
 		if (this.state.name.length === 0) {
-			console.log('name cannot be empty'); // eslint-disable-line
+			this.props.navigation.goBack();
 			return;
 		}
 
-		Meteor.call('inventorylists.create', this.state.name);
-		
-		this.setState({ name: '' });
-		this.props.navigation.goBack();
+		Meteor.call('inventorylists.create', this.state.name, (err, newListId) => {
+			if (err) {
+				Alert.alert(
+					"Error Creating New List",
+					err.error,
+					[
+						{ text: "OK", style: 'normal'}
+					],
+					{ cancelable: true }
+				);
+			}
+			
+			this.props.navigation.goBack();
```

Comment:
Can we navigate directly to the list after creating it?

Reply:
Ah sorry, just saw the comments after this. I'm fine with leaving that for a separate task.